src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1272,6 +1272,17 @@ Please specify one of them using the `--source` option to proceed.</value>
     <value>package has a version number that cannot be determined. Using "--include-unknown" may show more results.</value>
     <comment>{Locked="--include-unknown"} This string is preceded by a (integer) number of packages that do not have notated versions.</comment>
   </data>
+  <data name="UpgradeUnknownVersionCount" xml:space="preserve">
+    <value>package(s) have version numbers that cannot be determined. Use "--include-unknown" to see all results.</value>

```

Comment:
@AmelBawa-msft is working to fix our lack of placeholders in localization strings.  This family of strings will need to be updated and this comment is mostly a note for him.